PNA/agilentN5242A.py

```diff
@@ -285,8 +288,27 @@ def freq_sweep(self, c, log=False):
         # wait for sweep to finish
         sparams = yield self.getSweepData(dev, c['meas'])
         returnValue((freq*units.Hz, sparams))
-        
-        
+
+    @setting(102, returns='*v[Hz]*c')
+    def get_trace(self, c, trace='S21'):
+        """Get whatever trace is on the screen.
```

Comment:
I don't know what happens if you have e.g. trace S21 off and try to get it through this setting. What I meant to convey with "on the screen" is "whatever frequency/power/etc conditions are already set up (and you might need to have the trace you want activated, but I'm not sure)".